service/controller/azuremachinepool/handler/nodepool/template/main.json

```diff
@@ -185,7 +205,8 @@
         "gs-azure-operator.giantswarm.io-version": "[parameters('azureOperatorVersion')]",
         "kubernetes-version": "[parameters('kubernetesVersion')]",
         "min": "[int(parameters('minReplicas'))]",
-        "max": "[int(parameters('maxReplicas'))]"
+        "max": "[int(parameters('maxReplicas'))]",
+        "spot": "[if(parameters('spotInstancesEnabled'), 'true', 'false')]"
```

Comment:
I think this line can be removed

Reply:
Which line?

Reply:
oh it's a tag, I thought it was a parameter. Never mind.